Multisig/UI/Assets/AssetsView.swift

```diff
@@ -12,14 +12,33 @@ struct AssetsView: View {
     @FetchRequest(fetchRequest: Safe.fetchRequest().selected())
     var selectedSafe: FetchedResults<Safe>
 
+    @State var selection: Int? = 0
+
     var body: some View {
-        ZStack {
-            if selectedSafe.first != nil {
-                CoinsView(safe: selectedSafe.first!)
-            } else {
-                AddSafeIntroView()
+        TopTabView($selection) {
+            ZStack {
+                if selectedSafe.first != nil {
+                    BalancesView(safe: selectedSafe.first!)
+                } else {
+                    AddSafeIntroView()
+                }
+            }
+            .gnoTabItem(id: 0) {
+                HStack {
+                    Image("ico-coins")
+                    Text("COINS").font(Font.gnoCaption1)
+                }
```

Comment:
```suggestion
                }
                .frame(maxWidth: .infinity)
```

Reply:
funny that if you make the same for collectibles, it shrinks